docs/src/pages/components/switches/CustomizedSwitches.tsx

```diff
@@ -93,7 +92,6 @@ const AntSwitch = withStyles(theme => ({
     padding: 2,
     color: theme.palette.grey[500],
     '&$checked': {
-      transform: 'translateX(12px)',
```

Comment:
Oh nice, happy to see that we can remove some style :D

Reply:
Yeah, it is much easier now ;)